src/Cascade.php

```diff
@@ -31,6 +31,33 @@ class Cascade
      * @var Config
      */
     protected static $config = null;
+    
+    /**
+     * Inherit on undefined logger option
+     *
+     * @var bool
+     */
+    protected static $inherit_on_undefined_logger = false;
+
+    /**
+     * Getter for option `inherit_on_undefined_logger`
+     */
+    public static function shouldInheritOnUndefined()
```

Comment:
This is a nit, but traditionally this would be called `getInheritOnUndefinedLogger()` or, because the name implies it is a bool, you could just call is `inheritOnUndefinedLogger()`.